oshi-core/src/test/java/oshi/software/os/OperatingSystemTest.java

```diff
@@ -79,23 +75,11 @@ public void testOSProcessSetters(){
         OperatingSystem os = si.getOperatingSystem();
         OSProcess oldProcess = os.getProcess(os.getProcessId());
 
-        AbstractProcess newProcess = new AbstractProcess();
-        newProcess.setPath(oldProcess.getPath());
-        newProcess.setBytesRead(oldProcess.getBytesRead());
-        newProcess.setBytesWritten(oldProcess.getBytesWritten());
-        newProcess.setKernelTime(oldProcess.getKernelTime());
-        newProcess.setName(oldProcess.getName());
-        newProcess.setParentProcessID(oldProcess.getParentProcessID());
-        newProcess.setThreadCount(oldProcess.getThreadCount());
-        newProcess.setPriority(oldProcess.getPriority());
-        newProcess.setProcessID(oldProcess.getProcessID());
-        newProcess.setResidentSetSize(oldProcess.getResidentSetSize());
-        newProcess.setStartTime(oldProcess.getStartTime());
-        newProcess.setState(oldProcess.getState());
-        newProcess.setUpTime(oldProcess.getUpTime());
-        newProcess.setUserTime(oldProcess.getUserTime());
-        newProcess.setVirtualSize(oldProcess.getVirtualSize());
-        newProcess.setBytesWritten(oldProcess.getBytesWritten());
+        DefaultAbstractProcess newProcess = new DefaultAbstractProcess(oldProcess.getName(),oldProcess.getPath(),

```

Comment:
Why are we doing this?  We're removing tests of the setters, going in the opposite direction of what we're explicitly adding for Memory.    

And a more general question, other than a minor change (`abstract`), why are we mucking around with OSProcess in a PR for memory?